src/app/content/search/components/SearchResultsSidebar/index.spec.tsx

```diff
@@ -175,4 +178,30 @@ describe('SearchResultsSidebar', () => {
 
     expect(dispatch).toHaveBeenCalledWith(clearSearch());
   });
+
+  it('search sidebar header is visible on every new search', () => {
+
+    const {tree} = renderToDom(render());
+    store.dispatch(requestSearch('cool search'));
+    store.dispatch(
+      receiveSearchResults(
+        makeSearchResults([makeSearchResultHit({ book: archiveBook, page })])
+      )
+    );
+
+    const sidebar = ReactTestUtils.findRenderedComponentWithType(tree, SearchResultsBarWrapper);
+    const searchSidebar = sidebar.searchSidebar.current;
+    const header = sidebar.searchSidebarHeader.current;
+    const scrollSidebarSectionIntoViewMock = jest.spyOn(domUtils, 'scrollSidebarSectionIntoView');
+
+    expect(scrollSidebarSectionIntoViewMock).not.toHaveBeenCalled();
```

Comment:
since you just made this spy, it definitely won't have been called yet, you can probably remove this assertion